bedrock/settings/base.py

```diff
@@ -433,16 +433,13 @@ def language_url_map_with_fallbacks():
     # from redirects.urls
     "media",
     "static",
-    "certs",  # Is this still used?
-    "images",  # In redirects only
-    "contribute.json",  # served from root_files I think
+    "certs",
+    "images",  # root_files
```

Comment:
FYI these are still "present" (not sure about "used", but they're there… you'd want to see logs if they actually get any hits — there's an og mozilla-banner.gif, and two root certs: one expired and one still valid) so just updated the comments to reflect.